webpack.config.js

```diff
@@ -1,27 +1,59 @@
 const webpack = require('webpack');
-const {join} = require('path');
+
+class FallbackRequireWebpackPlugin {
+  apply(compiler) {
+	compiler.hooks.compilation.tap('FallbackRequireWebpackPlugin', (compilation) => {
+		const {renderRequire} = webpack.javascript.JavascriptModulesPlugin.getCompilationHooks(compilation);
+		renderRequire.tap(
+			'FallbackRequireWebpackPlugin',
+			(content, renderContext) => content.replace(
+				/\s*return module\.exports;\s*/,
+				'\nreturn (module.exports && module.exports.name === "webpackEmptyContext") ? require(moduleId) : module.exports;',
+			),
+		);
+	});
+  }
+}
```

Comment:
This plugin replaces packages required with webpack that don't match up with an existing module (has a `webpackEmptyContext` module generated), and allows fallback to the standard `require`. This is essential for requiring user-generated files and configs.

I will admit, this is not exactly pretty or documented, but it's better to patch Webpack than to patch many modules by themselves.

Reply:
> it's better to patch Webpack than to patch many modules by themselves.

I generally agree, although I'm still relatively unhappy with having to patch anything in particular (this isn't your fault - just griping at things in general)

How do we feel about the stability of this method? While not always true, my concern is "not documented" === "not stable API". I know you've done a lot of work to get to this point, it'd suck to have to go through this process of reading through Webpack's source every time there's a minor/patch update.

Also, are we able to turn this GH comment into a code one? Would be helpful for those in the future working on this file

Reply:
Added some comments! Thanks for the callout here